client/my-sites/checkout/checkout-thank-you/business-plan-details.jsx

```diff
@@ -72,11 +72,10 @@ const BusinessPlanDetails = ( {
 				icon={ <img alt="" src={ conciergeImage } /> }
 				title={ i18n.translate( 'Get personalized help' ) }
 				description={ i18n.translate(
-					'Schedule a Quick Start session with a Happiness Engineer to set up ' +
-						'your site and learn more about WordPress.com.'
+					'Schedule a $49 Quick Start session with a Happiness Engineer to set up your site and learn more about WordPress.com.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Schedule a Quick Start session with a Happiness Engineer to set up your site and learn more about WordPress.com.' )` *ES Score: 7*